src/App.js

```diff
@@ -57,12 +58,19 @@ const App = () => {
                 </ProtectedRoute>
               }
             />
+            <Route 
+              path="cohortsStudent"
+              element={
+                <ProtectedRoute>
+                  <CohortStudent />
```

Comment:
In the above sense it would be nice to have this component called Cohort and load data based on the student/teacher role